openwisp_radius/migrations/0001_initial_freeradius.py

```diff
@@ -69,7 +69,11 @@ class Migration(migrations.Migration):
                 ('terminate_cause', models.CharField(blank=True, db_column='acctterminatecause', max_length=32, null=True, verbose_name='termination cause')),
                 ('service_type', models.CharField(blank=True, db_column='servicetype', max_length=32, null=True, verbose_name='service type')),
                 ('framed_protocol', models.CharField(blank=True, db_column='framedprotocol', max_length=32, null=True, verbose_name='framed protocol')),
-                ('framed_ip_address', models.GenericIPAddressField(blank=True, db_column='framedipaddress', db_index=True, null=True, verbose_name='framed IP address')),
+                ('framed_ip_address', models.GenericIPAddressField(blank=True, db_column='framedipaddress', null=True, verbose_name='framed IP address')),
+                ('framed_ipv6_address', models.GenericIPAddressField(blank=True, db_column='framedipv6address', protocol='IPv6', null=True, verbose_name='framed IPv6 address')),
```

Comment:
Were these manually added or automatically created because some previous PR didn't create them. 
I am just a little unsure what's going on here and how does it relate to django 2.2 update!?

Reply:
I added it manually, I added it because of `django-freeradius`  dependency. Adding this is needed because of this recent commit https://github.com/openwisp/django-freeradius/commit/e6f39a669ae7a2ffecae663adb07b9cee82ead5f

Reply:
Please rever this change and rebase on the current master

Reply:
Branch was already rebased on current master and reverting this change leads to error

Reply:
@oltarzewskik no, no that problem you saw was found by @nemesisdesign and he fixed it so now this can be reverted after you rebase again to the master! :smile:

Reply:
Ignore me< think I am late, this is fixed, closing! :smile: